src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss

```diff
@@ -0,0 +1,8 @@
+.MyAccountPasswordForm-Button {
+    @include mobile {
+        width: 100%;
+    }
+    position: relative;
+    left: 50%;
+    transform: translateX(-50%);
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4724192)